extensions-core/orc-extensions/src/test/java/org/apache/druid/data/input/orc/OrcReaderTest.java

```diff
@@ -314,8 +314,13 @@ public void testJsonPathFunctions() throws IOException
         //deviation of [7,8,9] is 1/3, stddev is sqrt(1/3), approximately 0.8165
         Assert.assertEquals(0.8165, Double.parseDouble(Iterables.getOnlyElement(row.getDimension("stddev"))), 0.0001);
 
-        //append is not supported
-        Assert.assertEquals(Collections.emptyList(), row.getDimension("append"));
+        //Support for append has been added in json-path-2.9.0 but it is unused
+        Exception exception = Assert.assertThrows(UnsupportedOperationException.class, () -> {
+          row.getDimension("append");
+        });
+        String expectedMessage = "Unused";
+        String actualMessage = exception.getMessage();
+        Assert.assertTrue(actualMessage.contains(expectedMessage));
```

Comment:
do we know wha the full error message is that will get surfaced to the user if they use this functionality?

Reply:
can this be simplified to do assertEquals since `"Unused"` is the full exception message?

Reply:
```suggestion
        Assert.assertEquals("Unused", exception.getMessage());
```